lib/views/home/home_view.dart

```diff
@@ -155,7 +155,11 @@ class _HomeViewState extends State<HomeView> {
             children: [
               _text(topic: "Amount", value: items[index].amount),
               _text(topic: "Description", value: items[index].description),
-              _text(topic: "Bolt11", value: items[index].bolt11),
+              _text(
+                  topic: items[index].boltIdentifier == "bolt11"
+                      ? "Bolt11"
+                      : "Bolt12",
+                  value: items[index].bolt),
```

Comment:
```suggestion
                  topic: items[index].topic()",
                  value: items[index].bolt),
```
Add a new function `topic` that hides the if-else logic, do not put logic inside the UI code that it is not UI specific